network/mikrotik/snmp/mode/frequency.pm

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright 2018 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package network::mikrotik::snmp::mode::frequency;
```

Comment:
I do not see help, why checking this if we cannot set any thresholds ? Could you explain why you need to check this frequency plz ?

Reply:
Sure, this isn't a counter. We have over 800 appliances and we want to use Centreon Map. So instead of going into each appliance and manually checking the frequency of each wireless interface, we opted to make a service which we could just display within centreon map. Hope this clears things up.

Do you wish for us to add --help to this mode?